### PR TITLE
OpenSSH `9.9p1`

### LINKED ISSUE #3007: wanted: openssh
### Module location

https://www.openssl.org/

### Link to bzlmod issue in the module's repository

_No response_

### Any other context to provide?

OpenSSH project provides `ssh`, `scp` and `sftp` binaries. Having these can make it easier to do hermetic remote connection rules.

It has a dependency on a crypto library such as OpenSSL but we may be able to use `boringssl` which is already in the BCR.

### Fund our work

- [ ] Sponsor our community's open source work by donating a [feature bounty](https://opencollective.com/bazel-rules-authors-sig/projects/bazel-central-registry)

### PR DESCRIPTION
**Pull request by mattyclarkson**
Implements a native Bazel build around OpenSSH 9.9 portable.

Requires `libxcrypt` from #3025.

Fixes #3007

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (openssh) have been updated in this PR. Please review the changes.

**Comment by mattyclarkson** [comment]
@meteorcloudy this is a follow-up to #3025. Could I get a `presubmit-auto-run` label 🙇

File: modules/openssh/9.9p1/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+bcr_test_module:
+  module_path: ""
+  matrix:
+    bazel:
+      - 7.x
+      - 8.0.0rc1
```
Comment (meteorcloudy) [inline modules/openssh/9.9p1/presubmit.yml:6]: nit: 8.0.0rc2 is also out

**Review by meteorcloudy — DISMISSED** [review]

**Comment by mattyclarkson** [comment]
Needs #3047

> Looks like #3025 has some issues. Feel free to send a PR to update the MODULE.bazel file, we can still force merge it and since it's already broken, no one should be depending on it.

Sorry. #3047 resolves that.

**Pull request merged**